src/bzfs/bzfs.cxx

```diff
@@ -1942,25 +1942,35 @@ static void recalcHandicap(int playerIndex)
       relscore += me->player.howManyTimesKilledBy(i) - p->player.howManyTimesKilledBy(playerIndex);
     }
   }
+  
+  bz_ComputeHandiCap_V1 data;
+  data.playerID = playerIndex;
+  data.desiredHandicap = std::max(0, relscore);
+
+  worldEventManager.callEvents(bz_eComputeHandicapEvent, &data);
 
-  me->score.setHandicap(std::max(0,relscore));
+  me->score.setHandicap(data.desiredHandicap);
 }
 
 // calculate handicap values for all players
-static void recalcAllHandicaps()
+void recalcAllHandicaps()
 {
   if (!handicapAllowed())
     return;
 
+  worldEventManager.callEvents(&bz_EventData(bz_eBeginHandicapRefreshEvent));
+
   for (int i = 0; i < curMaxPlayers; i++) {
     GameKeeper::Player *p = GameKeeper::Player::getPlayerByIndex(i);
     if (p && realPlayer(i) && !p->player.isObserver())
       recalcHandicap(i);
   }
+
+  worldEventManager.callEvents(&bz_EventData(bz_eEndHandicapRefreshEvent));
```

Comment:
This doesn't compile for me. The `callEvents` expects two arguments. Same as the `bz_eBeginHandicapRefreshEvent` call